bin/prover-client/Cargo.toml

```diff
@@ -41,6 +41,7 @@ rayon = "1.8.0"
 reth-rpc-types = { workspace = true }
 rockbound = { workspace = true }
 serde = { workspace = true }
+serde_json = { workspace = true }
 sp1-sdk = "2.0.0"
 strata-tx-parser = { workspace = true }
 thiserror = { workspace = true }
```

Comment:
Can we change all the `foo = { workspace = true }` in this file for `foo = workspace.true`?